x-pack/plugins/xpack_main/server/lib/xpack_info.js

```diff
@@ -137,6 +155,14 @@ export class XPackInfo {
       }
 
       this._cache = { response };
+
+      if (licenseInfoChanged) {
```

Comment:
any reason why this is not in the same block that starts on 137?

Reply:
Yeah, the listeners could very well be calling methods on the `XpackInfo` instance that access the cache, like `isAvailable()` and `unavailableReason()` are in the xpack_main listener. I doubt there's a reason the cache needs to be updated after the log line, but I didn't want to change the order of things unnecessarily.

Reply:
ah, ok  - makes sense!